utils/directories.cc

```diff
@@ -140,7 +148,23 @@ future<> directories::do_verify_owner_and_mode(fs::path path, recursive recurse,
 };
 
 future<> directories::verify_owner_and_mode(fs::path path, recursive recursive) {
-    return do_verify_owner_and_mode(std::move(path), recursive, 0);
+    return do_verify_owner_and_mode(std::move(path), recursive, 0, [](fs::path &path){return true;});
```

Comment:
If the signature would use a std::function parameter it should be easy to define this as the default, for backward compatibility.
and nit: please sprinkle some spaces in the lambda definition so it be more readable (like around the parenthesis and inside the curly braces)

Reply:
Done. An empty function is now the default and `do_verify_owner_and_mode` now checks for empty function before executing it.